src/services/billing/cost-management/cost-dashboard/widgets/LastMonthTotalSpend.vue

```diff
@@ -0,0 +1,165 @@
+            categoryAxis.dataFields.category = categoryKey;
+            categoryAxis.renderer.minGridDistance = 1;
+            categoryAxis.renderer.grid.template.strokeOpacity = 0;
+            categoryAxis.renderer.grid.template.disabled = true;
+
+            categoryAxis.renderer.labels.template.adapter.add('text', (text, target) => {
+                if (target.dataItem.category === state.firstMonth.date) return state.firstMonth.date;
+                if (target.dataItem.category === state.lastMonth.date) return state.lastMonth.date;
+                return '';
+            });
+
+            const valueAxis = chart.yAxes.push(new am4charts.ValueAxis());
+            valueAxis.renderer.labels.template.disabled = true;
+            valueAxis.renderer.grid.template.strokeOpacity = 0;
+            valueAxis.renderer.baseGrid.disabled = true;
+            valueAxis.min = 0;
+
+            const series = chart.series.push(new am4charts.ColumnSeries());
+            series.dataFields.valueY = valueName;
+            series.dataFields.categoryX = categoryKey;
+            series.columns.template.propertyFields.fill = 'color';
+            series.columns.template.strokeOpacity = 0;
+            series.columns.template.column.cornerRadiusTopLeft = 6;
+            series.columns.template.column.cornerRadiusTopRight = 6;
+            series.columns.template.width = am4core.percent(45);
+            series.columns.template.tooltipText = '{categoryX}: [bold]{valueY}[/b]';
+
+            series.columns.template.adapter.add('fill', (fill, target) => {
+                if (target.dataItem?.dataContext?.date === state.thisMonth.date) {
+                    return am4core.color(violet[200]);
+                }
+                if (target.dataItem?.dataContext?.date === state.lastMonth.date) {
+                    return am4core.color(violet[500]);
+                }
+                return fill;
+            });
+            chart.data = state.data;
+        };
+
+        const getChartData = async () => {
+            try {
+                state.loading = true;
+                const { results } = await SpaceConnector.client.costAnalysis.cost.analyze({
+                    include_usage_quantity: false,
+                    granularity: 'MONTHLY',
+                    filter: [],
+                    pivot_type: 'CHART',
+                    start: '2020-11-01T00:00:00Z',
+                    end: '2021-10-16T00:00:00Z',
+                    page: {
+                        limit: 12,
+                    },
+                });
+                state.data = results.map(d => ({
+                    date: d.date,
+                    usd_cost: d.values[0].usd_cost,
+                }));
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                state.data = [];
+            } finally {
+                state.loading = false;
+            }
+        };
+
+        (() => {
+            getChartData();
+        })();
```

Comment:
```suggestion
            getChartData();
```

getChartData가 모두 실행이 끝나고 화면이 그려져야 하는게 아니면, 제안드린 방식으로만 해도 좋을 것 같습니다!

Reply:
로딩 끝나기 전에 데이터를 그리려고 하면 undefined 에러가 나서 ㅠㅠ 차트 로딩이랑 데이터 로딩이랑 분리해볼게요!!